app/Http/Requests/AccountOptionRequest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class AccountOptionRequest extends FormRequest
```

Comment:
We should not extend the default `FormRequest`cof Laravel, You can see, we've our own `FormRequest` by modifying the error response in a fashionable way, we need to use that.

Extend this `FormRequest` -
https://github.com/arhanashik/PSTUian-web-backend/blob/main/app/Http/Requests/FormRequest.php


You can check DonationRequest class.